### ISSUE TITLE
Add function to copy/fill in (T)OTP passwords to browser addon

### ISSUE THREAD

**Issue by ushuc** [kee-org/browser-addon#335]
KeePass supports OTP/TOTP (One Time Password / Timebased One Time Password) functionality for two factor authentication. When a website I want to log in to requests this TOTP, I have to go to the KeePass program to copy the (T)OTP. It would be great if the Kee browser plugin could show, copy and fill in this (T)OTP, like it does with regular passwords.

![grafik](https://github.com/kee-org/browser-addon/assets/109863013/8fb8eae9-a433-4bc6-8419-667236ed2c5b)

**Comment by CennoxX** [comment]
It may not be as easy as for password and username, but it is already possible to copy and fill TOTPs:
You've to go the `Kee`-tab of the entry, then to `Form fields`, where you can add any value you want, including fields like `{TIMEOTP}`. There you can add the html `id` and/or `name` attribute of the TOTP field (in GitHubs case it is app_otp and app_totp).
![grafik](https://github.com/kee-org/browser-addon/assets/6975881/d49c4fc0-d388-4eae-a377-a20f63e135de)

All fields you add above are shown in the addon:
![grafik](https://github.com/kee-org/browser-addon/assets/6975881/07a047b0-5f4b-4dfa-93dc-4adfb72c9970)

The text field name or ID must be also filled in the addon settings (1) > (2) > Finding forms > Include List > Text field name/ID so they can be filled automatically.

**Comment by CennoxX** [comment]
@ushuc It seems you don't have placeholders for the browser addon enabled. For this, you'll have to open KeePass > File > Database Settings… > Kee > KeePass placeholders > activate Enable

**Comment by luckyrat** [comment]
Yeah you should only enable Placeholders on the specific fields you require rather than the entire database, but if you already have hundreds from much older databases, the "Database Settings" option can be a good temporary measure. I'll fix the bug in v2 but won't promise to keep that database-wide feature around forever.

**Comment by CennoxX** [comment]
Using KeePass with stored TOTP does not inherently compromise the principle of two-factor authentication, It just redefines the factors involved. The knowledge factor is now represented by the KeePass password, while the possession factor is no longer the authenticated phone but access to the KeePass database file and access to the KeePass key file.
